group/assignment6/RearrrangingCars.java

```diff
@@ -0,0 +1,71 @@
+package assignment_6;
+
+import java.util.Hashtable;
+import java.util.LinkedList;
+import java.util.Map.Entry;
+import java.util.Random;
+
+public class RearrrangingCars {
+
+	private Hashtable<Integer, Character> start;
+	private Hashtable<Integer, Character> end;
+	private Random rand;
+	private int noCarKey = 0;
+	private int noExistKey = -1;

```

Comment:
Probably should make `noCarKey` and `noExistKey`  `final`.

Reply:
i changed noCarKey to final as i deleted noExistKey to throw an error instead